README.md

```diff
@@ -59,7 +59,7 @@ class BgColor {
   }
 }
 
-customAttributes.define('bg-color', BgColor);
+customAttributes.default.define('bg-color', BgColor);
```

Comment:
Hm, yeah I don't want to have to do this... Will look for a way around.

Reply:
Yea, curious if you have any thoughts. Moving to ESM in your examples would definitely move people in that direction, but I didn’t see anything in rollup that lets you define your UMD module as an actual function